packages/valory/skills/decision_maker_abci/behaviours/decision_receive.py

```diff
@@ -445,6 +457,7 @@ def async_act(self) -> Generator:
                 vote,
                 confidence,
                 bet_amount,
+                weighted_accuracy,
```

Comment:
The weighted accuracy is stored in the synchronized data but this is not necessary and also utilized in the same round, therefore, before it is set:

https://github.com/valory-xyz/trader/blob/720ffecb17459274610c0286a03c754542c04a57/packages/valory/skills/decision_maker_abci/behaviours/decision_receive.py#L373

Reply:
Which action should I do here?

Reply:
ok I think I got the idea. Fixing this in the next commit